### PR TITLE
Add mnemonics

### LINKED ISSUE #11: [feature request] add a mnemonic field
- show it under the hint
- maybe rename “hint” to “category”
- have a setting to show and hide it

### PR DESCRIPTION
**Pull request by notwaldorf**
Mnemonics are super useful for hiragana/katakana.
Fixes #11 

Looks like this:
screen shot 2018-05-15 at 8 52 30 am

### CODE REVIEW

File: src/reducers/data.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ const stats = (state = {}, action) => {
   }
 }
 
-const byHint = (state = {}, action) => {
+const byCategory = (state = {}, action) => {
```
Comment (frankiefu) [inline src/reducers/data.js]: https://github.com/notwaldorf/flash-cards/blob/bddf6cfa68893fe265c5e0cf177f9ca10d01f9a6/src/reducers/data.js#L52 is still referencing `byHint`

Comment (frankiefu) [reply src/reducers/data.js]: After making this one change locally I am still not seeing mnemonics.
screen shot 2018-05-15 at 10 59 29 am

Comment (notwaldorf) [reply src/reducers/data.js]: oh hold on i think my rebase went poorly

Comment (frankiefu) [reply src/reducers/data.js]: ok np

**Comment by notwaldorf** [comment]
thanks!!

**Pull request merged**